products/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from products.views import ProductCategoryView, ProductListView
+
+urlpatterns = [
+    # http://127.0.0.1:8000/products/categories/1
+    path('/categories/<int:category_id>', ProductCategoryView.as_view()),
```

Comment:
굳이 적어주지 않아도 되는 주석인것 같아요!
```suggestion
    path('/categories/<int:category_id>', ProductCategoryView.as_view()),
```

Reply:
늘 꼼꼼히 리뷰해주셔서 정말 감사드립니다!
해당 주석은 삭제 처리 하겠습니다.